src/common.browser/MicAudioSource.ts

```diff
@@ -78,6 +78,12 @@ export class MicAudioSource implements IAudioSource {
         return PromiseHelper.fromResult(MicAudioSource.AUDIOFORMAT);
     }
 
+    public get blob(): Promise<Blob> {
+        const deferred = new Deferred<Blob>();
+        deferred.reject("Not implemented for Mic input");
+        return deferred.promise();
```

Comment:
There should be a PromiseHelper.reject (or something close)

Reply:
PromiseHelper.fromError, it looks like

Reply:
Corrected